### ISSUE TITLE
armv5 support?

### ISSUE THREAD

**Issue by PieterSten** [philippe44/AirConnect#65]
Hi, 

would it be possible to use airupnp on a Synology box with ARMv5 processor? The ARMv7 build doesn't work there.

thanks for any help and keep up the great work!

**Comment by philippe44** [comment]
I've uploaded an arm5 version, but I cannot really test it. let me know if it works

**Comment by PieterSten** [comment]
yep, that seems to work! I'll do some more tests and let you know if I find anything.

Thanks for your great work!

**Comment by alewin17** [comment]
Hi, 
I have the same issue that ARMv7 is not working, where can I find the link to the ARMv5 version?
Thanks in advance for your great work!

**Comment by philippe44** [comment]
With all other binaries. It’s named -arm5

**Comment by emanut** [comment]
> I've uploaded an arm5 version, but I cannot really test it. let me know if it works

is this somewhere available? Thanks
